src/language_concepts/mutability.md

```diff
@@ -43,6 +43,63 @@ fn helper(mut x: i32) {
 }
 ```
 
+### Constants
+
+A constant type is a value that does not change per circuit instance. This is different to a witness which changes per proof. If a constant type that is being used in your program is changed, then your circuit will also change.
+
+Below we show how to declare a constant value:
+
+```rust,noplaypen
+fn main() {
+    let a: comptime Field = 5;
+
+    // `comptime Field` can also be inferred:
```

Comment:
Unlike the other sections, I'm fine with this note on inference here as "comptime" is likely something that will be unfamiliar to most users and they may incorrectly assume it is not part of the type and thus cannot be inferred otherwise.

Reply:
Thanks, will keep the suggestion in mind 🙌